src/proxy/router.js

```diff
@@ -93,6 +95,10 @@ export default class Router {
         return false;
     }
 
+    _processStaticContent () {
+
```

Comment:
Add comment here: `Do nothing =)`. So it will be clear what this is a virtual method and not piece of abandoned code. Smile at the end is the absolute semantic requirement in this sentence, BTW :hamburger: